### DISCUSSION TITLE
Changing volume of specific wallpaper through the command line.

### DISCUSSION THREAD

**Discussion by Ucef27** [rocksdanister/lively#365] [Q&A] [answered]
Hello I am wondering how to change the volume of a specific wallpaper through the command line.

**Comment by rocksdanister** [comment, marked as answer]
Not possible directly, you can mute it if its video:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#customize-wallpaper

`setprop --monitor 1 --property "mute=true"`

**Reply by Ucef27** [reply]
Thank you!
